src/main/java/org/openstreetmap/atlas/utilities/command/abstractcommand/AbstractAtlasShellToolsCommand.java

```diff
@@ -784,15 +783,23 @@ private void printSimpleUsageMenu()
         printlnStderr(builder.toString());
     }
 
-    private String[] unpackTailSentinelArguments(final String[] args)
+    private String[] unpackSentinelArguments(final String[] args)
     {
-        // check the last arg to see if we should check for other tail arguments
-        if (args.length > 0 && JAVA_MARKER_SENTINEL.equals(args[args.length - 1]))
+        List<String> argsAsList = Arrays.asList(args);
+        argsAsList = new ArrayList<>(argsAsList);
+        if (!argsAsList.isEmpty() && IGNORE_UNKNOWN_OPTIONS.equals(argsAsList.get(0)))
```

Comment:
Why are we limiting this to being the first argument?

Reply:
We can change it so this code scans all of ARGV instead. I had it this way just to make it clear that it is a hack. Options are supposed to be handled by `SimpleOptionAndArgumentParser` but I didn't want to clutter that class with this particular workaround, as it is supposed to be for more general use.